src/viur/core/render/html/env/viur.py

```diff
@@ -727,3 +730,8 @@ def seoUrlForEntry(render: Render, *args, **kwargs):
 @jinjaGlobalFunction
 def seoUrlToFunction(render: Render, *args, **kwargs):
     return utils.seoUrlToFunction(*args, **kwargs)
+
+
+@jinjaGlobalFunction
+def otp_uri_to_qrcode(render: Render, uri: str, *args, **kwargs) -> str:
```

Comment:
Such a function is useful. Can we make in more general, so any data can be encoded as a QR-code?